### ISSUE TITLE
L2 norm or L1 norm of the netwrok weights as a regularization in the loss function?

### ISSUE THREAD

**Issue by zhaogengyan** [junyanz/pytorch-CycleGAN-and-pix2pix#625]
For the pix2pix network, is there a L2 norm or L1 norm of the netwrok weights as a regularization in the loss function in the current version of code? Thanks a lot.

**Comment by junyanz** [comment]
We didn't use weight decay. You can add it by yourself.

**Comment by HareshMiriyala** [comment]
i dont think L1 or L2 norm is implemented. the loss used in this implementation for pix2pix is BCEWithLogitsLoss which only considers binary cross entropy with logits.
